src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html

```diff
@@ -32,7 +30,7 @@
           <tr class="catchment-water-quality-table-footer">
               <td></td>
```

Comment:
We can expand the table cell by using a `colspan` attribute so the label is not so squished.  If you delete the empty `<td></td>` and then add `colspan="2"` to the "Total For Area..." cell, it'll look like this:

![screenshot from 2016-11-01 17 45 14](https://cloud.githubusercontent.com/assets/1014341/19908738/f459a52e-a05a-11e6-8295-054f7d4a7397.png)